quickevent/app/quickevent/plugins/Runs/src/runsplugin.cpp

```diff
@@ -546,10 +546,19 @@ qf::core::utils::TreeTable RunsPlugin::stageResultsTable(int stage_id, const QSt
 		qf::core::sql::QueryBuilder qb;
 		qb.select2("classes", "id, name")
 			.select2("courses", "id, length, climb")
+			.select("runnersFinished, runnersCount")
 			.from("classes")
 			//.where("classes.name NOT IN ('D21B', 'H40B', 'H35C', 'H55B')")
 			.joinRestricted("classes.id", "classdefs.classId", "classdefs.stageId={{stage_id}}")
 			.join("classdefs.courseId", "courses.id")
+			.join("JOIN (SELECT COUNT(runs.competitorId) AS runnersCount, "
```

Comment:
umi toto i SQLite?

Reply:
Na SQLite mi to chodí, PostgreSQL by to snad mělo taky bez problému zvládnout

Reply:
PostgreSQL je taky OK